Packs/CortexXDR/Integrations/XQLQueryingEngine/XQLQueryingEngine.yml

```diff
@@ -1967,15 +1637,12 @@ script:
     - contextPath: PaloAltoNetworksXQL.ProcessCausalityNetworkActivity.results._product
       description: The result product.
       type: String
+    description: Search for network connection created by a given process causality
+      ID.
+    polling: true
   dockerimage: demisto/python3:3.9.8.24399
-  feed: false
-  isfetch: false
-  longRunning: false
-  longRunningPort: false
   runonce: false
-  script: '-'
   subtype: python3
-  type: python
+fromversion: 5.5.0
 tests:
 - Cortex XDR - XQL Query - Test
-fromversion: 5.5.0
```

Comment:
Why the whole file was changed? did you execute `demisto-sdk format` on it? I think it is better if we keep it like the previous format and only change the fields which are relevant to your change (time_frame argument).